### PR TITLE
Refactor send transaction

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

File: src/screens/SendCoinsConfirmScreen.tsx [hunk 1]
```diff
@@ -114,30 +112,20 @@ class SendCoinsConfirmScreen extends Component<Props> {
             type: MessageType.success,
             buttonProps: {
               title: i18n.message.returnToDashboard,
-              onPress: () => this.props.navigation.navigate(Route.MainCardStackNavigator),
+              onPress: () => navigation.navigate(Route.MainCardStackNavigator),
```
Comment (Kamill90) [inline src/screens/SendCoinsConfirmScreen.tsx:115]: please move it to class method

File: src/screens/SendCoinsConfirmScreen.tsx [hunk 2]
```diff
@@ -114,30 +112,20 @@ class SendCoinsConfirmScreen extends Component<Props> {
...
-          {
-            text: 'OK',
-            onPress: () => this.props.navigation.goBack(),
-          },
-        ]);
-      }
-    });
+        },
+        onFailure: (error: string) => {
+          Alert.alert('ERROR', error, [
```
Comment (Kamill90) [inline src/screens/SendCoinsConfirmScreen.tsx:120]: should not it be from locales?

**Review by mateuszpmroz — APPROVED** [review]

**Review by Kamill90 — APPROVED** [review]

File: src/screens/SendCoinsConfirmScreen.tsx (hunk 1 above)
Comment (KedziaPawel) [reply src/screens/SendCoinsConfirmScreen.tsx:115]: will be fixed in the optimize wallets fetching PR

File: src/screens/SendCoinsConfirmScreen.tsx (hunk 2 above)
Comment (KedziaPawel) [reply src/screens/SendCoinsConfirmScreen.tsx:120]: do you mean this "ERROR"?

**Pull request merged**